src/main.js

```diff
@@ -31,6 +32,16 @@ function handleUnitSwitchClick(e) {
   updateWeather(weatherData, unitSystem);
 }
 
+async function handleDisplayWeatherBySearch(event) {
+  let searchText = $("#input").value;
+  let splitText = searchText.split(",");
+  splitText = splitText.map((string) => string.trim()).join(",");
+  let coords = await getLocationCoords(splitText);
+
+  weatherData = await getWeatherDetails(coords[0], coords[1]);
```

Comment:
Let's make it more readable by using [Destructuring assignment ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).

```suggestion
  let [lat, lon] = await getLocationCoords(splitText);

  weatherData = await getWeatherDetails(lat, lon);
```